### PR TITLE
Removed most of my code ownership

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
Apologies for not using the template 😄 . I get pinged on discussions, issues, and PRs for these packages but I either don't use them anymore or don't have context+time to be of use.

### CODE REVIEW

**Review by peterblazejewicz — CHANGES_REQUESTED** [review]
Please revert changes in `.github/CODEOWNERS`, it's generated file.
Removing from headers is enough

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@JoshuaKGoldberg thanks for your contributions!

**Comment by JoshuaKGoldberg** [comment]
Lol...

File: types/jest-environment-puppeteer/index.d.ts [hunk 1]
```diff
@@ -1,7 +1,6 @@
 // Type definitions for jest-environment-puppeteer 4.4
 // Project: https://github.com/smooth-code/jest-puppeteer/tree/master/packages/jest-environment-puppeteer
-// Definitions by: Josh Goldberg <https://github.com/joshuakgoldberg>
-//                 Ifiok Jr. <https://github.com/ifiokjr>
+// Definitions by: Ifiok Jr. <https://github.com/ifiokjr>
```
Comment (peterblazejewicz) [inline types/jest-environment-puppeteer/index.d.ts:3]: can you revert this file change? Package is outdated and blocks CI

**Review by peterblazejewicz — APPROVED** [review]

**Comment by JoshuaKGoldberg** [comment]
Ready to merge

**Pull request merged**